bottles/backend/wine/winecommand.py

```diff
@@ -601,6 +603,21 @@ def get_cmd(
 
         if pre_script is not None:
             command = f"sh '{pre_script}' ; {command}"
+        
+        if midi_soundfont is not None:
+            command = f"""
+                # start fluidsynth in dedicated terminal
+                # (as, for some reason, fluidsynth doesn't like being run with &)
+                fluidsynth="fluidsynth --server --audio-driver=pipewire '{midi_soundfont}'"
+                flatpak-spawn --host gnome-terminal -- bash -c "$fluidsynth"
```

Comment:
As stated, `fluidsynth` doesn't seem to work when run in the background (see: [https://www.reddit.com/r/Lutris/comments/i0g25z/comment/hwo880k/](https://www.reddit.com/r/Lutris/comments/i0g25z/comment/hwo880k/). As such, a dedicated separate terminal window seem to be the next good (and working) option.

Naturally, `gnome-terminal` is but a placeholder. Bottles go-to choice seems to be easyterm.py. I assume there's also a way for the latter to run a separate command as intended (and maybe hide the window? That would be perfect but not sure if possible).

Reply:
Resolved by using `pyfluidsynth` (Python bindings for the `libfluidsynth` API). No more calling commands or terminal nonsense.